ui/app/templates/vault/cluster/clients/index.hbs

```diff
@@ -62,5 +63,6 @@
   </Toolbar>
   <Clients::Config @model={{this.model.config}} @isLoading={{this.currentlyLoading}} />
 {{else}}
+  {{!-- <Clients::Dashboard @tab={{this.tab}} @model={{this.model}} @isLoading={{this.currentlyLoading}} /> --}}
```

Comment:
Is this still needed?

Reply:
Yeah, this is the hidden code. It's how users can access the 1.9 client counts while we work on 1.10 which has no data coming in.